src/buffer/mod.rs

```diff
@@ -26,7 +26,7 @@
 
 pub use drm_fourcc::{DrmFourcc, DrmModifier, DrmVendor, UnrecognizedFourcc, UnrecognizedVendor};
 
-/// A handle to a GEM buffer.
+/// A handle to a GEM buffer
```

Comment:
Not really sure about line endings. It seems function docs are commonly sentences ended with a period, but type docs are generally left open...